File: app/src/main/java/com/rv150/musictransfer/utils/UiThread.java
package com.rv150.musictransfer.utils;

import android.os.Handler;
import android.os.Looper;

public class UiThread {
    private static final Handler HANDLER = new Handler(Looper.getMainLooper());

    public static void run(Runnable runnable) {
        HANDLER.post(runnable);
    }
}

File: app/src/main/java/com/rv150/musictransfer/utils/UiThread.kt
package com.rv150.musictransfer.utils

import android.os.Handler
import android.os.Looper

object UiThread {
    private val HANDLER = Handler(Looper.getMainLooper())

    fun run(runnable: Runnable) {
        HANDLER.post(runnable)
    }
}